src/pages/portfolio/index.tsx

```diff
@@ -77,43 +78,61 @@ const useStyles = makeStyles((theme) => ({
 
 const PortfolioPage: FC<{ data: ProjectGQL }> = ({ data }) => {
   const classes = useStyles();
-  const defaultType = 'All';
   const projectData = data.portfolioPage.frontmatter;
   const { projects } = projectData;
   const { componentType, isMobile } = useComponentType();
   const developerProfile = useDeveloperProfile();
   const [selectedProject, setSelectedProject] = useState(-1);
-  const [navbarTitle, setNavbarTitle] = useState(0);
-  const projectLabels = [defaultType, ...new Set(projects.map((project) => project.projectLabel))];
-
+  const [navbarNumberTitle, setNavbarNumberTitle] = useState(0);
+  const sortedGroupsOfProjects = getTabsData(projects);
+  const projectsLabels: string[] = [...sortedGroupsOfProjects.map((object) => object.projectLabel)];
   const getNavbarTitle = (type: number) => {
-    const title: Record<number, string> = { ...projectLabels };
+    const title = projectsLabels;
     return title[type];
   };
+  const tabLabelTitle = getNavbarTitle(navbarNumberTitle);
 
-  const projectType = getNavbarTitle(navbarTitle);
-
-  const filteredProjects = projects.filter((project) => {
-    return projectType !== defaultType ? project.projectLabel === projectType : ' ';
-  });
+  // filter all projects depending on their label
+  const filteredProjects = sortedGroupsOfProjects
+    .filter((project) => {
+      return project.projectLabel === tabLabelTitle;
+    })
+    .map((project) => project.projectProperties)
+    .flat(1);
 
   const handleChange: TabsProps['onChange'] = (event, newValue) => {
-    setNavbarTitle(newValue);
+    setNavbarNumberTitle(newValue);
+  };
+
+  const previousNavbarTitle = () => {
+    setNavbarNumberTitle(navbarNumberTitle === 0 ? projectsLabels.length - 1 : navbarNumberTitle - 1);
+    setSelectedProject(
+      navbarNumberTitle === 0
+        ? sortedGroupsOfProjects[projectsLabels.length - 1].projectProperties.length - 1
+        : sortedGroupsOfProjects[navbarNumberTitle - 1].projectProperties.length - 1,
+    );
+  };
+
+  const nextNavbarTitle = () => {
+    setNavbarNumberTitle(navbarNumberTitle === projectsLabels.length - 1 ? 0 : navbarNumberTitle + 1);
+    setSelectedProject(0);
   };
 
   // no project will have index equal to -1 therefore no project will be selected
   const handleCloseProject = () => {
     setSelectedProject(-1);
   };
 
-  // if is first project, choose last project
+  // choose previous project or if is first label, choose last label
```

Comment:
In my opinion, the name of the function is really meaningful and we don't need a comment here